onnxruntime/core/optimizer/compute_optimizer/upstream_reshape_actors.cc

```diff
@@ -282,6 +283,23 @@ bool LayerNormalizationReshapeActor::PreCheck(
   return propagate_input_indices.size() > 0;
 }
 
+bool LayerNormalizationReshapeActor::PostProcess(
+    Graph& /* graph */, Node& current_node, const ReshapeInfo& /* info_without_node */,
+    const logging::Logger& /* logger */,
+    std::vector<int>& /* propagate_input_indices */,
+    const std::unordered_map<int, std::vector<DimCompare>>& /* all_input_cmp_rets */,
+    const std::unordered_map<int, ReshapeInfo>& /* new_reshape_infos */) {
+  auto axis = static_cast<int64_t>(current_node.GetAttributes().at("axis").i());
```

Comment:
update the axis to positive number to handling axis=-1, -2, etc?

Reply:
In UpstreamReshape for Layernorm senario the axis is only possible to be -1, 2, other case will check fail and return before.
So, here we check axis > 1 actually only for the axis = 2 case. When axis = -1 , we need not update it.